hypar-io/Elements
# Add AnalysisImage class

BACKGROUND:
- AnalysisMesh only supports a limited number of vertices when converted to glb. We could have extended this serialization, but given that analyses may end up becoming quite large, opted for another class that would apply analysis colors via a texture image rather than mesh vertices.

DESCRIPTION:
- Adds AnalysisImage class and test
- Adds `InterpolateTexture` member to `Material`, which if false will serialize the material so that pixels render as hard edges rather than blurred interpolation. This is true by default so that the image behavior will not change on existing code.
- Small thing: add vscode settings to gitignore so that local user settings do not pollute the repository.

TESTING:
- Run the AnalysisImage test
- Try using AnalysisImage in your function
- See https://hypar.io/workflows/66091bef-6cf7-4253-96a5-69ae8e3c99d4 for an example of AnalysisImage in use
  
FUTURE WORK:
- N/A

REQUIRED:
- [x] All changes are up to date in `CHANGELOG.md`.

COMMENTS:
- N/A


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/hypar-io/elements/575)
